packages/components/Persona/src/Persona.tokens.root.ts

```diff
@@ -1,11 +1,12 @@
-import { ViewStyle, ViewProps } from 'react-native';
+import { ViewStyle } from 'react-native';
 import { styleFunction } from '@uifabricshared/foundation-tokens';
 import { IPersonaTokens } from './Persona.types';
 import { ITheme } from '@uifabricshared/theming-ramp';
+import type { IViewProps } from '@fluentui-react-native/adapters';
 
 const _rootKeyProps: (keyof IPersonaTokens)[] = ['coinSize', 'size'];
 
-function _buildRootStyle(tokenProps: IPersonaTokens): ViewProps {
+function _buildRootStyle(tokenProps: IPersonaTokens): IViewProps {
   const rootStyle: ViewStyle = {
```

Comment:
Is the expectation that all controls use IViewProps instead of ViewProps from React Native?

Reply:
I think so - the adapters package filters out the necessary props for each separate platform, so I don't see the negative to it. Although I'm open to hear others' thoughts.

Reply:
I don't see a negative to it either... I suppose we were using ViewProps in some cases because there wasn't a need for platform specific props but I don't see an issue changing it over.

Reply:
I mostly asked so I could update contributing guidance / understand the ramifications of that. I think we've avoided doing that on the apple side because IViewProps doesn't include macOS and at the time none of us guide understood what it meant. That, and Jason told me we were moving away from the "I" prefix so I just kinda filtered those out of my head 😅

Reply:
Yeah, the "I" prefix may have been have been a habit carried over from other programming languages? Perhaps we should create a issue about it for discussion.